doc/conf.py

```diff
@@ -80,12 +83,13 @@
     "nbsphinx",
     "sphinx_autosummary_accessors",
     "sphinx.ext.linkcode",
-    "sphinx_panels",
     "sphinxext.opengraph",
     "sphinx_copybutton",
     "sphinxext.rediraffe",
+    "sphinx_design",
```

Comment:
we should update these pages to use `sphinx-design` instead of `sphinx-panels`

- https://github.com/pydata/xarray/blob/main/doc/team.rst
- https://github.com/pydata/xarray/blob/main/doc/tutorials-and-videos.rst

Reply:
hmmm. I think you are right. It seems some of the css settings conflict when trying to use both. I'll try to get back to this soon but others should feel free to take a swing at these pages before me if they like.

Reply:
@jhamman, I pushed new changes with fixes... let me know if I missed anything